.vsts-ci.yml

```diff
@@ -15,134 +15,141 @@ pr:
     include:
     - '*'
 
-jobs:
-- template: /eng/common/templates/jobs/jobs.yml
-  parameters:
-    enableMicrobuild: true
-    enablePublishBuildArtifacts: true
-    enablePublishTestResults: true
-    enablePublishBuildAssets: true
-    enableTelemetry: true
-    helixRepo: dotnet/command-line-api
-    timeoutInMinutes: 180 # increase timeout since BAR publishing might wait a long time
-    jobs:
-    - job: Windows
-      pool:
-        # For public or PR jobs, use the hosted pool.  For internal jobs use the internal pool.
-        # Will eventually change this to two BYOC pools.
-        ${{ if ne(variables['System.TeamProject'], 'internal') }}:
-          name: NetCorePublic-Pool
-          queue: BuildPool.Windows.10.Amd64.VS2019.Open
-        ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-          name: NetCoreInternal-Pool
-          queue: BuildPool.Windows.10.Amd64.VS2019
-      variables:
-      # Only enable publishing in official builds.
-      - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-        # DotNet-Blob-Feed provides: dotnetfeed-storage-access-key-1
-        # Publish-Build-Assets provides: MaestroAccessToken, BotAccount-dotnet-maestro-bot-PAT
-        - group: DotNet-Blob-Feed
-        - group: DotNet-Symbol-Server-Pats
-        - group: Publish-Build-Assets
-        - name: _OfficialBuildArgs
-          value: /p:DotNetSignType=$(_SignType)
-                 /p:TeamName=$(_TeamName)
-                 /p:DotNetPublishBlobFeedKey=$(dotnetfeed-storage-access-key-1)
-                 /p:DotNetPublishBlobFeedUrl=https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
-                 /p:DotNetPublishToBlobFeed=$(_DotNetPublishToBlobFeed)
-                 /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
-                 /p:PublishToSymbolServer=true
-                 /p:DotNetSymbolServerTokenMsdl=$(microsoft-symbol-server-pat)
-                 /p:DotNetSymbolServerTokenSymWeb=$(symweb-symbol-server-pat)
-      # else
-      - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
-        - name: _OfficialBuildArgs
-          value: ''
-      strategy:
-        matrix:
-          ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
-            Debug:
-              _BuildConfig: Debug
-              _SignType: test
-              _DotNetPublishToBlobFeed: false
-              _BuildArgs:
+stages:
+- stage: build
+  displayName: Build and Test
+  jobs:
+  - template: /eng/common/templates/jobs/jobs.yml
+    parameters:
```

Comment:
You need to pass `enablePublishUsingPipelines: true` here. See reference here: https://github.com/dotnet/arcade/pull/4556/files?short_path=5153417#diff-c88223c5b65116dd5625673d42760adfR77